skins/cat17/templates/Membership_Application.html.twig

```diff
@@ -45,6 +45,14 @@
 							'showActiveMembershipBenefits': showMembershipTypeOption == 'true'
 						} only %}
 					</div>
+					<div class="col-xs-12 col-md-4">
+						<div class="external-link">
+							<a href="https://wikimedia.de/wiki/Mitgliedschaft" target="_blank">
```

Comment:
Not wrong, but I'd make it "configurable" in the content repo (dedicated key).

Reply:
Originally, I advised Tonina against configurability because I imagined the link to be only in one place and putting it in `app/config` would have felt wrong to me. But since it will also appear on the donation confirmation page and could hypothetically have an English version, I've changed my mind and also think that we could put this into `messages.json`.